packages/commonwealth/client/scripts/views/components/CommunityStake/useCommunityStake.ts

```diff
@@ -99,6 +99,7 @@ const useCommunityStake = (props: UseCommunityStakeProps = {}) => {
     isLoading,
     activeChainId,
     refetchStakeQuery,
+    featureFlagEnabled: communityStakeEnabled,
```

Comment:
lets export it as the same name `communityStakeEnabled` for clarity.

Reply:
Ah sorry, this should be removed, thanks for the catch